src/BloomExe/Book/BookStorage.cs

```diff
@@ -866,6 +866,8 @@ private string SanitizeNameForFileSystem(string name)
 				name = name.Replace(c, ' ');
 			}
 			name = name.Trim();
+			if (name.StartsWith("."))
```

Comment:
I think your approach would work.  Do you want me to abort my pull request and have you submit a pull request, or do you want me to revise my pull request along those lines?  Your approach has the advantage that if the title consists only of dangerous characters, a valid filename is produced.